src/sass/core/_typography.scss

```diff
@@ -249,7 +249,7 @@ h1, h2, h3, h4, h5,
 
 h1, .h1 {
   @extend %Graphik-Regular-Web;
-  color: $typeAccent;
+  color: #444;
```

Comment:
Do we want to define this color as a variable to add a little context?

Reply:
@momo-sa I agree, using sass variable would make more sense here. Change made in the recent commit `9af4e21`.